Alignment/OfflineValidation/plugins/TrackerOfflineValidation.cc

```diff
@@ -2020,28 +2020,26 @@ std::pair<float, float> TrackerOfflineValidation::fitResiduals(TH1* hist) const
   float mean = hist->GetMean();
   float sigma = hist->GetRMS();
 
-  try {  // for < CMSSW_2_2_0 since ROOT warnings from fit are converted to exceptions
```

Comment:
@cms-sw/core-l2 can you confirm this?

Reply:
Confirm what exactly? We generally convert ROOT warnings to exceptions (with some predefined exceptions).

Reply:
The comment seems to imply the contrary, i.e. that in older releases the warnings are converted to exceptions while in newer it's not the case.

Reply:
We just had this issue https://github.com/cms-sw/cmssw/issues/42979 because of fit minimizer issuing a warning.

Reply:
Then I gather this change is undesireable